subset/ipaddr/dhcp_tests.py

```diff
@@ -79,6 +82,32 @@ def _test_dhcp_short():
         fd.close()
         return 'fail', 'No DHCP request received.'
 
+    def _test_ip_change():
```

Comment:
There should be an integration test to test the output of this script.

Reply:
> There should be an integration test to test the output of this script.

@henry54809 Is that not what device 05 in test_dhcp is for? Otherwise, are you suggesting a test_aux case?

Reply:
That integration only runs this function but it doesn't verify the output of the function. There should be something to check for generated reports after the run finishes in the existing dhcp integration test.

Reply:
@henry54809 Does the latest commit seem like what you had expected?